docs/README.md

```diff
@@ -114,6 +135,21 @@ spec:
       - kube-system
 ```
 
+### Resource Status
+All of the Sail Operator API resources have a `status` subresource that contains information about their current state in the Kubernetes cluster.
+
+#### Conditions
+All resources have a `Ready` condition which is set to `true` as soon as all child resource have been created and are deemed Ready by their respective controllers. To see additional conditions for each of the resources, check the [API reference documentation](https://github.com/istio-ecosystem/sail-operator/tree/main/docs/api-reference/sailoperator.io.md).
+
+#### InUse Detection
```

Comment:
Could you please add a few sentences when this is useful from user point of view?

Reply:
Ah, ok, it's used in the example below. So maybe just one sentence: "See the example bellow for proper usage."